samples/monster_generated.h

```diff
@@ -617,13 +617,14 @@ inline flatbuffers::TypeTable *ColorTypeTable() {
   static flatbuffers::TypeFunction type_refs[] = {
     ColorTypeTable
   };
+  static const int32_t values[] = { 0, 1, 2 };
```

Comment:
These previously were not generated if the values are equal to their indices, and now they are. Please check what is causing that.